test_schemas/en/test_plural_forms.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
As the plural count can come from a number answer we should use that for the test schema rather than have to create list collectors etc. We could have something as simple as one question asking for a number of things and the next question confirming it.